.travis.yml

```diff
@@ -1,16 +1,27 @@
+sudo: false
 language: python
 env:
-  - REQUESTS_VERSION=2.1.0
-  - REQUESTS_VERSION=2.8.1
-  - REQUESTS_VERSION=2.9.1
-python:
-  - "2.6"
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-install:
-  - if [[ $TRAVIS_PYTHON_VERSION == 2.6* ]]; then export PYTEST_ADDOPTS='-p no:pylint'; fi
-  - pip install -r requirements.txt
-  - pip install --upgrade --force-reinstall requests==$REQUESTS_VERSION
-script: py.test
+    - TOXENV=py27-requests
```

Comment:
The time taken to run all tests is directly correlated to the number of environments defined.

In this pull-request I chose to set all the possible tox environment variations. It is up to the maintainers of the project to decide exactly which environments to use. I can definitely reduce this to a more manageable set of environments.

I agree it is less clear identify which test version failed. The alternative is that the test environment definition is kept in only one file, the tox config file. And will be identical for both local and remote test runs.